params/protocol_params.go

```diff
@@ -188,17 +190,20 @@ const (
 )
 
 const (
-	DefaultBlockGenerationInterval    = int64(1) // unit: seconds
-	DefaultBlockGenerationTimeLimit   = 250 * time.Millisecond
-	DefaultOpcodeComputationCostLimit = uint64(100000000)
+	DefaultBlockGenerationInterval  = int64(1) // unit: seconds
+	DefaultBlockGenerationTimeLimit = 250 * time.Millisecond
+
+	OpcodeComputationCostLimitInfinite = math.MaxUint64
+	OpcodeComputationCostLimitOverride = uint64(0)
 )
 
 var (
 	// Dummy Randao fields to be used in a Randao-enabled Genesis block.
 	ZeroRandomReveal = make([]byte, 96)
 	ZeroMixHash      = make([]byte, 32)
 
-	TxGasHumanReadable uint64 = 4000000000 // NOTE: HumanReadable related functions are inactivated now
+	TxGasHumanReadable                   uint64 = 4000000000 // NOTE: HumanReadable related functions are inactivated now
+	ExperimentOpcodeComputationCostLimit        = OpcodeComputationCostLimitOverride
```

Comment:
```suggestion
	TxGasHumanReadable uint64 = 4000000000 // NOTE: HumanReadable related functions are inactivated now
```

Reply:
Changed the name of this value to suggested one. https://github.com/klaytn/klaytn/pull/2037/commits/3709fd8a22b92ef8553e5d422e216d8fc0436e99